# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
joint_limits:
  cylinder_joint:
    has_velocity_limits: true
    max_velocity: 6.159937255
    has_acceleration_limits: false
    max_acceleration: 0
  head_bearing_link:
    has_velocity_limits: true
    max_velocity: 6.159937255
    has_acceleration_limits: false
    max_acceleration: 0
  head_to_lamp:
    has_velocity_limits: true
    max_velocity: 6.159937255
    has_acceleration_limits: false
    max_acceleration: 0
  low_joint0:
    has_velocity_limits: true
    max_velocity: 6.159937255
    has_acceleration_limits: false
    max_acceleration: 0
  low_joint2:
    has_velocity_limits: true
    max_velocity: 6.159937255
    has_acceleration_limits: false
    max_acceleration: 0
  up_joint0:
    has_velocity_limits: true
    max_velocity: 6.159937255
    has_acceleration_limits: false
    max_acceleration: 0
  up_joint2:
    has_velocity_limits: true
    max_velocity: 6.159937255
    has_acceleration_limits: false
    max_acceleration: 0